extensions/amp-subscriptions/0.1/test/test-crypto-handler.js

```diff
@@ -131,6 +133,15 @@ describes.realWin(
           expect(cryptoSection2.textContent).to.equal(decryptedContent);
         });
       });
+
+      it('should fail due to key hashes being unequal', () => {
```

Comment:
If a hash is required in all cases we should test that missing hash does indeed fail and if it's not required in the publisher origin case we should that that a missing hash doesn't fail.

Reply:
Added tests for cases where we don't have a hash and do, and also when the hashes don't equal each other.